packages/fast-tooling-react/src/form/form-section.tsx

```diff
@@ -393,7 +407,17 @@ class FormSection extends React.Component<
         return null;
     }
 
-    private renderFormSection(): React.ReactNode {
+    private renderFormValidation(invalidMessage: string): React.ReactNode {
+        if (invalidMessage !== "") {
+            return (
+                <div className={this.props.managedClasses.formSection_invalidMessage}>
```

Comment:
should we be using the `classNames` utility?

Reply:
there's no reason when it's just a single class name and there is no logic for determining whether to add/remove it.